### PR TITLE
improve explanations of parameters for execute Relay Call guide

### PR DESCRIPTION
**Pull request by CJ42**
Screenshot 2023-10-06 at 17 32 56

Screenshot 2023-10-06 at 17 33 03

### CODE REVIEW

**Comment by frozeman** [comment]
I think for the validity timestamp it would be good to have a short summary of how the timestamp is constructed:
`uinte256<fromTimestamp> + unit12(toTimestamp), e.g.: .....`

also what's the `nonce nb` ?

**Comment by frozeman** [comment]
Also the signWith inter validator should ideally have comment, that explains how that's constructed. so people don't have to look into the code.

File: docs/guides/key-manager/execute-relay-transactions.md [hunk 1]
```diff
@@ -249,13 +249,22 @@ For more information check: [**How to sign relay transactions?**](../../standard
...
-  { value: chainId, type: 'uint256' },
-  { value: nonce, type: 'uint256' },
-  { value: validityTimestamps, type: 'uint256' },
-  { value: msgValue, type: 'uint256' },
-  { value: abiPayload, type: 'bytes' },
+  // MUST be number `25`
+  { value: LSP25_VERSION, type: 'uint256' },      // `0x0000000000000000000000000000000000000000000000000000000000000019`
+  // e.g: `4201` for LUKSO Testnet
+  { value: chainId, type: 'uint256' },            // `0x0000000000000000000000000000000000000000000000000000000000001069`
+  // e.g: nonce nb 5
```
[the remark this answers is not in the document]
Comment (magalimorin18) [reply docs/guides/key-manager/execute-relay-transactions.md:258]: And maybe we can add an explanation to which signing key this is ?

**Review by magalimorin18 — APPROVED** [review]

File: docs/guides/key-manager/execute-relay-transactions.md [hunk 2]
```diff
@@ -249,13 +249,22 @@ For more information check: [**How to sign relay transactions?**](../../standard
...
+  { value: nonce, type: 'uint256' },              // `0x0000000000000000000000000000000000000000000000000000000000000005`
+  // e.g: valid until 1st January 2025 at midnight (GMT). 
+  // Timestamp = 1735689600
```
[the remark this answers is not in the document]
Comment (magalimorin18) [reply docs/guides/key-manager/execute-relay-transactions.md:261]: https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-25-ExecuteRelayCall.md#:~:text=uint256%20channel)%20function.-,validityTimestamps%3A%20Two%20uint128%20timestamps%20concatenated%20together.%20The%20first%20timestamp%20determines%20from,point%20in%20time%20and%20the%20checks%20for%20the%20timestamps%20are%20skipped.,-payload%3A%20The%20abi

[the remark this answers is not in the document]
Comment (CJ42) [reply docs/guides/key-manager/execute-relay-transactions.md]: Created in a separate page in the "Contracts" section

**Pull request merged**